### ISSUE TITLE
json-ld data in html message

### ISSUE THREAD

**Issue by Maasik** [phpList/phplist3#840]
Need support for data format  "json-ld" - https://developers.google.com/gmail/markup/reference/formats/json-ld

`function disableJavascript($content) {
    ## disallow Javascript
    $content = str_ireplace('<script','< script',$content);`

disallows all script

**Comment by michield** [comment]
Yes, script tags are disabled for security reasons. Sending JS in an email is likely to make it be caught in spam filter. I can see that adding this json-ld can be useful, but I think that is best resolved with a plugin that automatically adds it to a campaign, and not by pasting the code in the campaign content part.
